.gitignore

```diff
@@ -29,6 +29,11 @@ local.properties
 # PDT-specific
 .buildpath
 
+#################
+## IDEA
+#################
+.idea/
+*.iml
```

Comment:
IDEA is not rouge, it's blue. Customizing the IDE to use a different color palette will likely confuse the auditing consultants. Please report to HR.